standards/containerisation_standards.md

```diff
@@ -0,0 +1,32 @@
+# Containerisation
+
+## We use Docker containers for delivery of all bespoke software being deployed to AWS and Azure
+Where we develop bespoke software, we default to using a Docker container as the deployable item.
+
+Unless the technology does not support containerisation, all deviations from this standard will be treated as architectural exceptions.
+
```

Comment:
Do we want to add a standard for how we orchestrate containers.  For example we use Kubernetes?

Reply:
Yes we do but we don't have one!

There are a bunch of additional standards that we will need, but I figured it's probably better to get this one down in writing (and get us all agreed on it) as a starting point.

Specifically, there's a bit of a difference of opinion on k8s on Azure vs AWS.  AKS seems viable but EKS not so much.

Reply:
I'd heard the same about EKS.  AKS also seems to have some limitations which mean it may not be suitable for all projects but it is incredibly easy to get a cluster up and running and let Microsoft take care of the hard work.

In FFC we're exploring the possibility of hosting a highly available k8s cluster federated across  both AWS (not EKS) and Azure.  I'll let you know how we get on.

Reply:
As an update to my comment, we tried it, it didn't work :-p.  Federation isn't really production ready yet so each FFC service will now be in a single cloud.  Maybe once federation has matured we can review again.

Also interestingly through our spikes we have agreed that managed Kubernetes clusters (ie AKS and EKS) are very much preferential to a IaaS approach.  I realise this is probably slightly off topic as I think others have commented that this standard should be limited to containers and not how they are later orchestrated and choreographed.